### ISSUE TITLE
Swift Package Manager support not working? 

### ISSUE THREAD

**Issue by pepejeria** [vtourraine/ThirdPartyMailer#21]
I tried adding the project using SPM in Xcode 11.5 and the following version will be suggested. 

<img width="734" alt="Screen Shot 2020-07-08 at 11 51 10" src="https://user-images.githubusercontent.com/1394550/86904664-52718e80-c111-11ea-9275-07fc15dbd8ec.png">

This is obviously the incorrect version and ends in an error.

**Comment by vtourraine** [comment]
Thanks a lot for the feedback! That’s something I noticed when I first added SPM support, but at first I thought that was just a weird bug. It’s still present, though, so we really need to address that.

I couldn’t find any documentation or online discussion about this specific issue. I’ve tried adding a `1.7.0` tag, that just points to the same commit as `1.7`, and it seems to solve the issue. 🤷 I’m not sure how SPM resolves version numbers, but I guess I’ll always pad with 3 numbers now (version X.Y.Z).

Anyway, hopefully that’s fixed now. Let me know how it goes.
